src/components/pages/ExploreDetail.jsx

```diff
@@ -157,9 +188,19 @@ class ExploreDetail extends React.Component {
           </div>
         </div>
         <div className="row similar-datasets-row">
-          <Title className="-secondary title">
-            Similar datasets
-          </Title>
+          <div className="column small-12">
+            <Title className="-secondary title">
+              Similar datasets
+            </Title>
+          </div>
+          <div className="column small-12">
+            <DatasetList
+              active={[]}
```

Comment:
Maybe instead of passing an active array we can set it as default property inside the DatasetList. What do you think?